bin/s2nd.c

```diff
@@ -189,6 +193,8 @@ void usage()
                     "    Ex: --psk psk_id,psk_secret,SHA256 --psk shared_id,shared_secret,SHA384.\n");
     fprintf(stderr, "  -E, --max-early-data \n");
     fprintf(stderr, "    Sets maximum early data allowed in session tickets. \n");
+    fprintf(stderr, "  -N --npn \n");
+    fprintf(stderr, "    Indicates support for the NPN extension. The '--alpn' option MUST be used with this option to signal the protocols supported."); 
```

Comment:
Is there a check for this? (in s2n_config_set_npn?)

Reply:
Eh, no. It's documented that you need to set the application protocols as well as call s2n_config_set_npn(). Plus, if you know enough about NPN to call s2n_config_set_npn, you should know that you have to set your supported protocols somewhere...